src/pages/tutorials/build-an-nft-marketplace/index.md

```diff
@@ -122,9 +134,9 @@ echo "{}" > esy.json
 
 **Your project is ready!**
 
-### FA2 contract
+### Step 2: FA2 contract
 
-We will rely on the Ligo FA library. To understand in detail how assets work on Tezos, please read below notes:
+Next, we will build the FA2 contract, which relies on the Ligo FA library. To understand in detail how assets work on Tezos, please read the notes below.
```

Comment:
"Please read the notes below" -- This is highly complicated info and will scare off tutorial readers. Summarize what the tutorial standard means in beginner-friendly terms or skip it.